OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -3206,4 +3212,7 @@ private static boolean isValidOutcomeEntry(String name) {
     * End OneSignalOutcome module
     */
 
+   interface OperationCompletedCallback {
```

Comment:
**Rename to one of these:**
`OSPromptCompletionCallback` or
`OSPromptActionCompletionCallback` or
`OSInAppMessagePromptCompletionCallback` or  
`OSInAppMessagePromptActionCallback`